src/jitter.rs

```diff
@@ -48,25 +48,53 @@ const MEMORY_SIZE: usize = MEMORY_BLOCKS * MEMORY_BLOCKSIZE;
 // Note: the C implementation relies on being compiled without optimizations.
 // This implementation goes through lengths to make the compiler not optimise
 // out what is technically dead code, but that does influence timing jitter.
-pub struct JitterRng {
+pub struct JitterRng<T: JitterTimer> {
     data: u64, // Actual random number
```

Comment:
As I understand it, outside of `gen_entropy` calls, this is either unused or the more significant half is used. So would it be better to move `data` to `EcState` and store an `Option<u32>` here?

Reply:
No, this is what is called the 'entropy pool' in previous versions of jitter_entropy. I wondered if it was necessary to preserve across runs. Suppose we initialize it with some constant with each round of `gen_entropy`. In theory, if the timer does not collect the expected number of bits of entropy per round once in a while, the result will be a little closer to that constant. Preserving `data` across runs will instead make it just a little closer to the previously generated value.